README.md

```diff
@@ -1,65 +1,78 @@
 
-# Component Detection
-![Component Detection CI](https://github.com/microsoft/component-detection/workflows/Component%20Detection%20CI/badge.svg)
-
-**For bugs, issues, and support please create an issue.**
-
-# Introduction
-
-ComponentDetection is a package scanning tool intended to be used at build time. CD produces a graph-based output of all detected components and supports a variety of open source package ecosystems.
-
-# Table of Contents
-
-* [Feature Overview](#Feature-Overview)
-* [My favorite language/ecosystem isn't supported!](#My-favorite-language/ecosystem-isn't-supported!)
-* [Building and running Component Detection](#Building-and-running-Component-Detection)
-    * [Running in Visual Studio (2019+)](#Running-in-Visual-Studio-(2019+))
-	* [Running from command line](#Running-from-command-line)
-	* [After building](#After-building)
-* [A detector is marked as DefaultOff/Experimental. What does that mean?](#A-detector-is-marked-as-DefaultOff/Experimental.-What-does-that-mean?)
-* [Telemetry](#Telemetry)
-
-# Feature Overview
-
-| Ecosystem | Scanning | Graph Creation |
-| - | - | - |
-| CocoaPods | ✔ | ✔ |
-| Linux (Debian, Alpine, Rhel, Centos, Fedora, Ubuntu)| ✔ (via [syft](https://github.com/anchore/syft)) | ❌ |
-| Gradle (lockfiles only) | ✔ | ❌ |
-| Go | ✔ | ❌ |
-| Maven | ✔ | ✔ |
-| NPM (including Yarn, Pnpm) | ✔ | ✔ |
-| NuGet (including Paket) | ✔ | ✔ |
-| Pip (Python) | ✔ | ✔ |
-| Poetry (Python, lockfiles only) | ✔ | ❌ |
-| Ruby | ✔ | ✔ |
-| Rust | ✔ | ✔ |
+<h1 align="center">
+  <br>
+  <a href="<What link>"><img src=".github/component-detection.png" alt="Component Detection" width="200"></a>
+  <br>
+  Component Detection
+  <br>
+</h1>
+
+<h4 align="center">Automatically detect the open-source libraries you use.</h4>
+
+<p align="center">
+    <img alt="Nuget" src="https://img.shields.io/nuget/v/Microsoft.ComponentDetection.Common">
+    <img alt="GitHub Workflow Status (with event)" src="https://github.com/microsoft/component-detection/actions/workflows/build.yml/badge.svg">
+    <img alt="GitHub CodeQL Status" src="https://github.com/microsoft/component-detection/actions/workflows/codeql-analysis.yml/badge.svg">
+    <img alt="OSSF-Scorecard Score" src="https://img.shields.io/ossf-scorecard/github.com/microsoft/component-detection">
+    <img alt="GitHub" src="https://img.shields.io/github/license/microsoft/component-detection">
```

Comment:
Can you explicitly make these links? GitHub makes badges clickable, but will take you to the `img` src. I'd prefer it if the NuGet badge took me to NuGet, the GitHub Actions badge took me to the action overview in GitHub, etc.